pathfinder/api/rest.py

```diff
@@ -43,6 +88,27 @@ def put(self):
                 str(err)
             )}, 400
 
+        # token_network_address and channel_id info is duplicate
+        # check, that both version match
+        if not token_network_address == balance_proof.token_network_address:
```

Comment:
`is_same_address` here? Both should be checksummed but I'd like to keep address comparisons consistent.